pyvista/examples/cells.py

```diff
@@ -986,6 +986,89 @@ def QuadraticQuadrilateral() -> UnstructuredGrid:
     return _make_isoparametric_unstructured_grid(_vtk.vtkQuadraticQuad())
 
 
+def QuadraticPolygon() -> UnstructuredGrid:
+    """Create a :class:`pyvista.UnstructuredGrid` containing a single quadratic polygon.
+
+    This cell corresponds to the :attr:`pyvista.CellType.QUADRATIC_POLYGON` cell type.
+
+    Returns
+    -------
+    pyvista.UnstructuredGrid
+        UnstructuredGrid containing a single quadratic polygon.
+
+    Examples
+    --------
+    Create and plot a single quadratic polygon.
+
+    >>> from pyvista import examples
+    >>> grid = examples.cells.QuadraticPolygon()
+    >>> examples.plot_cell(grid, cpos='xy')
+
+    List the grid's cells.
+
+    >>> grid.cells
+    array([8, 0, 1, 2, 3, 4, 5, 6, 7])
+
+    List the grid's points.
+
+    >>> grid.points
+    pyvista_ndarray([[0., 0., 0.],
+                     [2., 0., 0.],
+                     [2., 2., 0.],
+                     [0., 2., 0.],
+                     [1., 0., 0.],
+                     [3., 1., 0.],
+                     [1., 2., 0.],
+                     [0., 1., 0.]])
+
+    >>> grid.celltypes  # same as pyvista.CellType.QUADRATIC_POLYGON
+    array([36], dtype=uint8)
+    """
+    # TODO: Fix UnstructuredGrid so that this code correctly creates this cell:
+    # points = [
+    #     [0.0, 0.0, 0.0],
+    #     [2.0, 0.0, 0.0],
+    #     [2.0, 2.0, 0.0],
+    #     [0.0, 2.0, 0.0],
+    #     [1.0, 0.0, 0.0],
+    #     [2.0, 1.0, 0.0],
+    #     [1.0, 2.0, 0.0],
+    #     [0.0, 1.0, 0.0],
+    #     [3.0, 1.0, 0.0],
+    # ]
+    #
+    # cells = [len(points), *list(range(len(points)))]
+    # return UnstructuredGrid(cells, [CellType.QUADRATIC_POLYGON], points)
```

Comment:
Let's move this to an issue and add a link to the issue in the comments. I want to create an issue here, but you can create issues about this better than me in my mind.

Reply:
I found a bug with the example. They set 9 points but the cell only has 8. This resulted in 9 points incorrectly defined in the pyvista version. Fixed in https://github.com/pyvista/pyvista/pull/6596/commits/5cdda9dc797e09588e173edb4e50e1e2b72718c3